libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -218,6 +228,8 @@ TEST_GROUP_RUNNER( Defender_System )
      */
     RUN_TEST_CASE( Defender_System, Metrics_TCP_connections_all_are_published );
 
+    RUN_TEST_CASE( Defender_System, Metrics_TCP_connections_all_are_published_multiple_sockets );
```

Comment:
Would ```Metrics_Multiple_TCP_connections_all_are_published``` be a better name?

Reply:
I agree it's a better name but this has been the name for a long time. Might it affect other systems as well?